runtime/lib/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ if (TTMLIR_ENABLE_RUNTIME)
       ttnn/runtime.cpp
       ttnn/program.cpp
     )
+    target_compile_options(TTRuntimeTTNN PRIVATE -mavx -mavx2)
```

Comment:
Are these needed? `-mavx -mavx2`

Reply:
This is for converting data formats on host, apparently avx instructions are used to perform the conversions. The function using it is `::ttnn::to_dtype`

Reply:
I think this is part of metal compile though, we're linking against their precompiled dylib.  Perhaps one day we can add this if we do static linking + LTO